### ISSUE TITLE
Excel-like filter mode

### ISSUE THREAD

**Issue by nskiro** [react-bootstrap-table/react-bootstrap-table2#331]
Hi @AllenFang ,

Does table support excel-like filter mode?

**Comment by AllenFang** [comment]
@nskiro I dont think so, but you can check our live demo to see what filter we support: https://react-bootstrap-table.github.io/react-bootstrap-table2/storybook/index.html?selectedKind=Column%20Filter&selectedStory=Select%20Filter&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

please check the column filter section, thanks. and any idea is very welcome ;)
